src/Chapter2.hs

```diff
@@ -740,7 +752,8 @@ the list with only those lists that contain a passed element.
 
 🕯 HINT: Use the 'elem' function to check whether an element belongs to a list
 -}
-contains = error "contains: Not implemented!"
+contains :: Int -> [[Int]] -> [[Int]]
+contains x ll = filter (elem x) ll
```

Comment:
Correct solution and type!
Here you can also apply eta-reduction, to cut the last argument 🙂